src/app/screens/onboarding/detail-form/detail-form.component.html

```diff
@@ -0,0 +1,51 @@
+        <div class="absolute w-44 h-44 bg-purple-400 rounded-full shadow-lg transform scale-100 transition-transform duration-300 hover:scale-110" style="top: 40%; left: 10%; animation: pulse 1.5s infinite ease-in-out;"></div>
+        <div class="absolute w-50 h-50 bg-purple-400 rounded-full shadow-lg transform scale-100 transition-transform duration-300 hover:scale-110" style="top: 60%; left: 50%; animation: pulse 1.5s infinite ease-in-out;"></div>
+    </div>
+    <div class="relative z-10 bg-white bg-opacity-80 p-6 rounded-lg shadow-lg max-w-lg w-full">
+        <h1 class="text-3xl font-bold mb-4 text-center">Restaurant List</h1>
+        <form [formGroup]="restroDetails" (ngSubmit)="displayRestroDetails()">
+            <label class="block mb-2">Name:
+                <input id="restro-name" type="text" [formControl]="restroDetails.controls['name']" class="bg-rose-300 border-gray-300 p-2 rounded-lg w-full text-black" placeholder="Name">
+            </label>
+            <label class="block mb-2">Owner Name:
+                <input id="owner-name" type="text" [formControl]="restroDetails.controls['owner']" class="bg-rose-300 p-2 rounded-lg w-full" placeholder="Owner">
+            </label>
+            <label class="block mb-2">Type:
+                <input id="type" type="text" [formControl]="restroDetails.controls['type']" class="bg-rose-300 p-2 rounded-lg w-full" placeholder="Type (e.g., Dine-In, Takeaway)">
+            </label>
+            <label class="block mb-2">Contact Details:
+                <input id="contact" type="text" [formControl]="restroDetails.controls['contact']" class="bg-rose-300 p-2 rounded-lg w-full" placeholder="Contact Details">
+            </label>
+            <label class="block mb-2">Address:</label>
+            <label class="block mb-2">Street:
+                <input id="Street" type="text" [formControl]="restroDetails.controls['address'].controls['street']" class="bg-rose-300 p-2 rounded-lg w-full" placeholder="Street">
+            </label>
+            <label class="block mb-2">City:
+                <input id="city" type="text" [formControl]="restroDetails.controls['address'].controls['city']" class="bg-rose-300 p-2 rounded-lg w-full" placeholder="City">
+            </label>
+            <label class="block mb-2">Pin code:
+                <input id="zipcode" type="text" [formControl]="restroDetails.controls['address'].controls['zipcode']" class="bg-rose-300 p-2 rounded-lg w-full" placeholder="Pin Code">
+            </label>
+            <button type="submit" class="bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded-lg mt-4">Submit kar Do!</button>
+        </form>
+    </div>
+
+    <div *ngIf="submittedData" class="relative z-10 bg-white bg-opacity-80 p-6 rounded-lg shadow-lg max-w-lg w-full mt-6 animate-fadeIn">
```

Comment:
extract to a seperate component and practice data transfer between components

Reply:
yes sir done i create a child component and in that i have showed after submission details